### DISCUSSION TITLE
Screen shuts off after a few seconds

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#7046] [Q&A] [answered]
Hello,

Although the app is configured to "keep the screen on", my Android phone still turns itself off after a few seconds of no use… which I don't want while riding my bike.

I found no obvious option in Settings > Display.

Any idea where else I could look?

FWIW, I'm using the EvolutionX custom ROM, and OM 2023.12.21-1-Google (ie. the latest and greatest).

Thank you.

![image](https://github.com/organicmaps/organicmaps/assets/1541861/8f92e95c-cf65-4ad3-ab52-a36cf3152049)

**Comment by biodranik** [comment, marked as answer]
The fix https://github.com/organicmaps/organicmaps/pull/6674 was already merged and awaits for a new release.
